form-sharing/workflow.json

```diff
@@ -10,7 +10,7 @@
         "adaptor": "kobotoolbox",
         "expression": "1-getNewKoboForms.js",
         "next": {
-          "getSheetsList": "!state.error && state.data.length !== 0"
```

Comment:
@hunterachieng why removed the checking for `state.data.length !== 0` ?

Reply:
@hunterachieng yeah what is the thinking here?

Reply:
Since I do this in every job to handle the bulk state, `state.data={}` there will be no data there when checking

Reply:
If this change is for v1 then the changes in `workflow.json` doesn't matter but for v2 i think the edge path to down stream jobs should be something like this 
- For `1-getNewKoboForms.js` should be 
`!state.errors && state.koboForms.length > 0` and 
- For `2-getSheetsList.js` should be `!state.errors && state.filteredKoboFormsData.length > 0`